releasenotes/notes/fix-psycopg-async-iteration-d48a62f8aa597457.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes issue of psycopg async iterator previously not returning objects, by implementing async generator.
```

nit: We usually put the scope of the change at the front of the release note (so name of integration / feature / product / etc). That way user's can quickly scan our release notes and know what applies to them.

```suggestion
    psycopg: Fixes issue of psycopg async iterator previously not returning objects, by implementing async generator.
```